pkg/stanza/fileconsumer/file.go

```diff
@@ -58,6 +59,11 @@ func (m *Manager) Start(persister operator.Persister) error {
 			m.readerFactory.FromBeginning = true
 			m.tracker.LoadMetadata(offsets)
 		}
+		if m.pollsToArchive > 0 {
+			m.tracker.EnableArchiving(persister)
+		}
+	} else if m.pollsToArchive > 0 {
+		return fmt.Errorf("archiving is not supported in memory, please use a storage extension")
```

Comment:
Returning an error during Start will crash the entire collector. I think we either need to just log an error, or add this to the polls we remember in memory. For now, I would just log an error and we can worry about more complicated solutions later if needed.

Reply:
That works. Thanks!

Reply:
Done.